tacacs+.spec

```diff
@@ -31,6 +31,7 @@ export DONT_STRIP=1
 %{__rm} -rf %{buildroot}
 %makeinstall
 %{__install} -Dp -m0755 tac_plus.sysvinit %{buildroot}%{_initrddir}/tac_plus
+%{__install} -Dp -m0644 tac_plus.service %{buildroot}/etc/systemd/system/tac_plus.service
```

Comment:
You should use the RPM macros for doing this, it's more reliable and will guarantee the service ends up in the right place and gets registered properly. See the end of https://fedoraproject.org/wiki/Packaging:Systemd and https://fedoraproject.org/wiki/Packaging:Scriptlets?rd=Packaging:ScriptletSnippets#Systemd for details on this.